### ISSUE TITLE
[v2.0] Release date ?

### ISSUE THREAD

**Issue by frco9** [Meteor-Community-Packages/meteor-roles#203]
@alanning @mitar, Any plans about making V2.0 available as a Release Candidate anytime soon ? 

Thanks !

**Comment by Pushplaybang** [comment]
please let us know! looks like devs been quiet on 2.0, is close to ready?  any timeline, or a list of ToDo would be great.  Would love to see it come to life.

**Comment by tzapu** [comment]
i m using 2.0 by manually placing it in packages, seems to work ok so far, i do home it will get released at some point.

it s such a great and helpful package

**Comment by alanning** [comment]
Thanks for the reminder.  @mitar did a great job with 2.0 and I haven't followed through on getting a release packaged up.  There were a few changes that I had requested before so I'll double check the status and hopefully be able to get a new release out by August 13th.

If you'd like to use it now, you can do as @tzapu suggested and put the relevant repo files in your local packages dir and manually add it to `.meteor/packages`.

**Comment by jerfowler** [comment]
Any word on 2.0?

**Comment by ay13** [comment]
Any update?

**Comment by tcastelli** [comment]
Yes would be nice to know if there are any blocking tests/features :)

**Comment by abate** [comment]
I've been using the v2.0 branch for a while now in packages/ . It would be nice to have a alpha release

**Comment by mitar** [comment]
I renamed `partition` to `scope` as part of #213. If anyone was using this branch, you will have to rename it in the database. Probably something like from MongoDB shell:

```
db.users.update({}, {$rename: {'roles.partition': 'roles.scope'}}, {multi: true});
```

**Comment by abate** [comment]
I'm not sure if this is done on pourpose, but the example directory contains a .meteor this gives an error while detecting the current meteor version : 

```
Application's Meteor version (1.2.1) is less than 1.3.1; please use ulexus/meteor:legacy
ksp_1      | Meteor source found in /src/app/packages/meteor-roles/examples/flow-router/.meteor
```

I've removed the .meteor directory and everything goes as planned ...

**Comment by mitar** [comment]
This gives an error in your app? I think this is something buggy in code you have. That directory should not interfere with your app.

**Comment by abate** [comment]
not an app error. it works perfectly while on dev. Maybe this is just a problem with the docker image I'm using ... sorry for the noise.

**Comment by Taakn** [comment]
For those of you that have been using V2, is there anything to worry about in particular? Is it stable enough for production? What would be the best way to install the package?

Thanks so much for your help

**Comment by tzapu** [comment]
hi, been using it for 3-4 months, have not ran into any issues myself. brilliant package

i have just checked it out into a top level `packages` folder, and then did `meteor add meteor-roles` which istalls it from there

looking forward to it being released

**Comment by Taakn** [comment]
Thanks a lot @tzapu I really appreciate your answer!

**Comment by Taakn** [comment]
Hi, actually this is not working well for me. I did clone into .meteor/packages with git clone -b v2.0 --single-branch git://github.com/alanning/meteor-roles, however meteor add alanning:roles installs from master and not from my local copy in packages (so I end up with version 1.2.15). Do you know what I could be doing incorrectly ? Thanks

**Comment by Taakn** [comment]
I've also tried meteor add meteor-roles including changing the name in package.js but that didn't work

**Comment by Taakn** [comment]
ok nevermind it seems to be working now. Had to create a packages folder in my app folder, clone then change the name inside package.js. Thanks

**Comment by tcastelli** [comment]
christmas release gift? :D

**Comment by SimonSimCity** [comment]
Any updates on the release-date?

**Comment by JeremyIglehart** [comment]
Happy Valentines Day!  Can I offer my help in releasing version 2.0?

**Comment by alexpriceonline** [comment]
7 months since last bump. Any updates?! 👍

**Comment by jfols** [comment]
I've published the `v2.0` branch as `modweb:roles` https://atmospherejs.com/modweb/roles

I had some issues with Galaxy and git submodules, so went ahead and published the 2.0 branch--works great!

**Comment by mitar** [comment]
This has been released now.
